WordPress/src/test/java/org/wordpress/android/ui/mysite/QuickStartRepositoryTest.kt

```diff
@@ -105,26 +110,59 @@ class QuickStartRepositoryTest : BaseUnitTest() {
     }
 
     @Test
-    fun `refresh shows completion message and removes card if all tasks of a same type have been completed`() = test {
-        initStore()
+    fun `given dynamic card enabled + same type tasks done, when refresh started, then completion msg shown`() =
+            test {
+                whenever(quickStartDynamicCardsFeatureConfig.isEnabled()).thenReturn(true)
+                initStore()
 
-        val completionMessage = "All tasks completed!"
+                triggerQSRefreshAfterSameTypeTasksAreComplete()
 
-        whenever(selectedSiteRepository.getSelectedSite()).thenReturn(site)
-        whenever(quickStartUtils.isEveryQuickStartTaskDoneForType(siteId, GROW)).thenReturn(true)
-        whenever(resourceProvider.getString(any())).thenReturn(completionMessage)
-        whenever(htmlCompat.fromHtml(completionMessage)).thenReturn(completionMessage)
+                assertThat(snackbars).containsOnly(SnackbarMessageHolder(UiStringText(ALL_TASKS_COMPLETED_MESSAGE)))
+            }
 
-        val task = PUBLISH_POST
-        quickStartRepository.setActiveTask(task)
-        quickStartRepository.completeTask(task)
-        quickStartRepository.refresh()
+    @Test
+    fun `given dynamic card disabled + same type tasks done, when refresh started, then completion msg not shown`() =
+            test {
```

Comment:
Minor (🔍): Consider adding the below mock setup explicitly, just like it is done for when it is `true` or even for when it is `false` on some of the other such tests:

`whenever(quickStartDynamicCardsFeatureConfig.isEnabled()).thenReturn(false)`

PS: The same applies to another test further below.

Reply:
Added it explicitly in 440dc86, looks more readable, and adds to consistency 🙏.